tests/components/minecraft_server/test_config_flow.py

```diff
@@ -144,16 +202,17 @@
             )
 
             assert result["type"] == RESULT_TYPE_CREATE_ENTRY
-            assert result["title"] == f"{USER_INPUT[CONF_HOST]}:{USER_INPUT[CONF_PORT]}"
+            assert result["title"] == USER_INPUT[CONF_HOST]
             assert result["data"][CONF_NAME] == USER_INPUT[CONF_NAME]
-            assert result["data"][CONF_HOST] == USER_INPUT[CONF_HOST]
-            assert result["data"][CONF_PORT] == USER_INPUT[CONF_PORT]
+            assert result["data"][CONF_HOST] == "mc.dummyserver.com"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5232399)